portfolio/src/main/webapp/script.js

```diff
@@ -135,23 +137,51 @@ async function loadMemes() {
     data.forEach(memeObject => {
         const memeDiv = document.createElement("div");
         const memeImg = document.createElement("img");
+        const icon = document.createElement("button");
         const memeDesc = document.createElement("p");
 
         memeImg.src = memeObject.url; 
         memeImg.classList.add("meme-image");
+        memeDesc.innerHTML = memeObject.desc
 
-        memeDiv.innerText = "[" + memeObject.timeStamp + "] " 
+        icon.innerHTML = '<i class="fa fa-trash-o"></i>';
+        icon.classList.add("trash");
+        icon.setAttribute( "onClick", 
+            'deleteThis("Meme", ' + memeObject.id + ')');
+        memeDiv.innerHTML = "[" + memeObject.timeStamp + "] " 
             + memeObject.author + ":";
-        memeDesc.innerHTML = memeObject.desc;
         memeDiv.appendChild(memeImg);
         memeDiv.appendChild(memeDesc);
+        memeDesc.appendChild(icon);
 
         memeDiv.classList.add("meme-div");
         memeContainer.appendChild(memeDiv);
 
     })
 }
 
+/** Delete this element. */
+async function deleteThis(elementType, elementId) {
+    let response = await fetch('/delete', {
```

Comment:
if you're not using response, you don't need to set it to a variable

Reply:
gotcha